src/test/java/seedu/address/logic/parser/InternshipDataParserTest.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.InternshipMessages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.InternshipMessages.MESSAGE_UNKNOWN_COMMAND;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.InternshipTypicalIndexes.INDEX_FIRST_INTERNSHIP;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.InternshipAddCommand;
+import seedu.address.logic.commands.InternshipClearCommand;
+import seedu.address.logic.commands.InternshipDeleteCommand;
+import seedu.address.logic.commands.InternshipEditCommand;
+import seedu.address.logic.commands.InternshipEditCommand.EditInternshipDescriptor;
+import seedu.address.logic.commands.InternshipExitCommand;
+import seedu.address.logic.commands.InternshipFindCommand;
+import seedu.address.logic.commands.InternshipHelpCommand;
+import seedu.address.logic.commands.InternshipListCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.internship.CompanyNameContainsKeywordsPredicate;
+import seedu.address.model.internship.Internship;
+import seedu.address.testutil.EditInternshipDescriptorBuilder;
+import seedu.address.testutil.InternshipBuilder;
+import seedu.address.testutil.InternshipUtil;
+
+public class InternshipDataParserTest {
+
+    private final InternshipDataParser parser = new InternshipDataParser();
+
+    @Test
+    public void parseCommand_add() throws Exception {
```

Comment:
Perhaps can be more specific with the Exception being thrown

Reply:
Actually, I think this is fine cause it's the same case for Person, they just generally put Exception. The test case ideally shouldn't throw an Exception because that's not what it is testing.